modules/management/kubectl-apply/variables.tf

```diff
@@ -41,15 +41,15 @@ variable "kueue" {
   description = "Install and configure [Kueue](https://kueue.sigs.k8s.io/docs/overview/) workload scheduler. A configuration yaml/template file can be provided with config_path to be applied right after kueue installation. If a template file provided, its variables can be set to config_template_vars."
   type = object({
     install              = optional(bool, false)
-    version              = optional(string, "v0.8.1")
+    version              = optional(string, "v0.9.0")
```

Comment:
Let's not update the default , but use the version we want from the blueprint

Reply:
Could you please explain why we should not update the default? My rationale for updating was that the changes should be backward compatible across versions and TAS support is independent of the node we choose.

Reply:
I agree with you on all versions being backward compatible and updating the default eventually @ighosh98. We should wait till we know for a fact that 0.9.0 is stable though